app/Http/Controllers/UserController.php

```diff
@@ -12,15 +14,19 @@ class UserController extends Controller
      */
     public function index()
     {
+        $users = User::with('tasks')->get();
+
         return view('users.index', [
-            'users' => User::all(),
+            'users' => $users,
         ]);
     }
 
     public function tasks(User $user)
```

Comment:
tên function nên để dạng động từ 
```suggestion
    public function getTasksByUser(User $user)
```